packages/blockchain-wallet-v4-frontend/src/data/modules/sfox/actionTypes.js

```diff
@@ -37,3 +37,8 @@ export const SFOX_SELL_BTC_PAYMENT_UPDATED_LOADING =
   '@COMPONENT.SFOX_SELL_BTC_PAYMENT_UPDATED_LOADING'
 export const SFOX_SELL_BTC_PAYMENT_UPDATED_FAILURE =
   '@COMPONENT.SFOX_SELL_BTC_PAYMENT_UPDATED_FAILURE'
+
+export const GET_JUMIO_TOKEN = '@COMPONENT.GET_JUMIO_TOKEN'
```

Comment:
we should structure actionType namespaces.
we discussed with Guillaume that there should be @EVENT namespace for 'events'.
The way I see it now nboth @EVENT and @COMPONENT namespaces are not exactly meaningful it just states that are used in `data/components`, so I don't feel it would be a useful namespace if big part of the code would reside in component.

I think we can distinguish between two big types of namespaces:
- @EVENT actions that trigger sagas and serve the perpose of event layer between views and controllers, this events are coupled to the components.
- Other namespace maybe @STATE, which triggers reducers and is coupled to them, and is more of a model layer.
These ones are purely architectural can be used to understand project, and don't other problems.

Other than that a module/component name should serve as a (sub)namespace and are required to distinguish between various identical actions such as SUBMIT_FORM_SUCCES, CREATE and others.
These are also vital for understanding which module acts by looking at devtools.